jopa-impl/pom.xml

```diff
@@ -56,6 +56,11 @@
             <artifactId>antlr4-runtime</artifactId>
             <version>${org.antlr4.version}</version>
         </dependency>
+        <dependency>
```

Comment:
What do you need this library for? I am trying to keep the dependencies of JOPA minimal, so I just want to know if it is strictly necessary.